Project.toml

```diff
@@ -4,6 +4,9 @@ authors = ["Andrew R. Winters <andrew.ross.winters@liu.se>", "Michael Schlottke-
 version = "0.1.0-pre"
 
 [deps]
+MuladdMacro = "46d2c3a1-f734-5fdb-9937-b9b9aeba4221"
+Static = "aedffcd0-7271-4cad-89d0-dc628f76c6d3"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
```

Comment:
These need [compat] bounds. You can just take the ones of Trixi.jl.

Reply:
Done in c02cbf60e0396aed8f64950577559dd3052b5a5c